tv/.env

```diff
@@ -1,3 +1,6 @@
-# CDP
+# CMP
 NEXT_PUBLIC_CMP_PREVIEW_ENDPOINT_URL=
-NEXT_PUBLIC_CMP_PREVIEW_API_KEY=
\ No newline at end of file
+NEXT_PUBLIC_CMP_PREVIEW_API_KEY=
+
+# Website
+NEXT_PUBLIC_WEBSITE_URL=
```

Comment:
Now that we have the Vercel variable, it is ok to have this variable set to the same value as in the kiosk .env file. It is used only for local development anyway. It is ok to commit the value for this one. It is not a secret.